README.md

```diff
@@ -3,7 +3,7 @@
 # `image uploads`
 
 ![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/dwyl/imgup/ci.yml?label=build&style=flat-square&branch=main)
-[![codecov.io](https://img.shields.io/codecov/c/github/dwyl/imgup/main.svg?style=flat-square)](https://codecov.io/github/dwyl/imgup?branch=main)
+[![codecov.io](https://img.shields.io/codecov/c/github/dwyl/imgup/main.svg?style=flat-square)](http://codecov.io/github/dwyl/imgup?branch=main)
```

Comment:
The https version _works_ though ... https://codecov.io/github/dwyl/imgup?branch=main
<img width="1159" alt="image" src="https://github.com/dwyl/imgup/assets/194400/0318a01d-03f5-4801-9df4-c4362f192d60">